runtime/ftplugin/help.lua

```diff
@@ -112,7 +112,15 @@ vim.keymap.set('n', 'g==', function()
   end
 end, { buffer = true })
 
+vim.bo.keywordprg = ':HelpKeywordPrg'
```

Comment:
does something like this work?
```suggestion
vim.bo.keywordprg = ":lua vim._ftplugin.help.keywordprg()"
```

Reply:
It works, but it will also raise an error `= expected near EOF` (which is quite annoying to users), because Nvim would try to run this
```vim
:lua vim._ftplugin.help.keywordprg() some_keyword
```

Reply:
#31280 can be related.